src/browser/api/browser_view.ts

```diff
@@ -0,0 +1,81 @@
+import { BrowserView, BrowserViewConstructorOptions, Rectangle, AutoResizeOptions, webContents } from 'electron';
+import { Identity } from '../api_protocol/transport_strategy/api_transport_base';
+import { addBrowserView, getBrowserViewByIdentity, getWindowByUuidName, OfView, removeBrowserView } from '../core_state';
+import { getRuntimeProxyWindow } from '../window_groups_runtime_proxy';
+import { BrowserViewOptions, BrowserViewCreationOptions } from '../../../js-adapter/src/api/browserview/browserview';
+import convertOptions = require('../convert_options');
+import {getInfo as getWebContentsInfo} from './webcontents';
+import of_events from '../of_events';
+import route from '../../common/route';
+
+
+const windowCloseListenerMap = new WeakMap();
+
+export interface BrowserViewOpts extends BrowserViewCreationOptions {
+    uuid: string;
+}
+
+export async function create(options: BrowserViewOpts) {
+    const targetWin = getWindowByUuidName(options.target.uuid, options.target.name);
```

Comment:
Does there need to be a check here to make sure target isn't `undefined`? Also should we destructure `options`?